test/functional/tests/install/test_install_rpm.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas import installer, rpm
+from api.cas.cas_module import CasModule
+from core.test_run import TestRun
+from test_utils import os_utils
+
+
+@pytest.mark.remote_only()
+@pytest.mark.uninstall_cas()
+def test_install_from_rpm():
+    """
+        title: Test for installing OpenCAS from RPM
+        description: |
+          Check if OpenCAS could be installed from RPM and then uninstalled.
+        pass_criteria:
+          - OpenCAS installs successfully
+          - OpenCAS uninstalls successfully
+    """
+    with TestRun.step("Download and build OpenCAS."):
+        installer.rsync_opencas(install_rpm=True)
+        installer.set_up_opencas()
```

Comment:
`set_up_opencas()` installs opencas from sources. Is this necessary during this test?